### ISSUE TITLE
should site location data be editable

### ISSUE THREAD

**Issue by jeffreyhanson** [NCC-CNC/whattodo#6]
@josephrbennett suggested the following:

> Do you think the basic site location data should be editable in tool 2, similar to all other aspects? Just in case there is an error with original file from GIS, and someone wants to quickly correct location in the tool itself? Not a big deal either way; just wondering.

**Comment by jeffreyhanson** [comment]
Yeah, that's a nice idea! It should be possible - but would be a fair of work. Once we've the core functionality of tool 2 working, let's revisit this and see if there's enough time/scope left to incorpperate this?

**Comment by ricschuster** [comment]
I like the idea too, but that does sound like a lot of work and I don't even know whether it should be part of a v2. Probably even later.

**Comment by jeffreyhanson** [comment]
Yeah, it could be a lot of work. It definitely wouldn't be feasible to let users add in new sites with the April deadline - but editing the geometries of existing sites might be possible depending on quickly I can get the initial working version done.

**Comment by josephrbennett** [comment]
OK, sorry - no worries, definitely a minor thing. Sorry for putting the issue in the wrong spot.
